images-list

```diff
@@ -33,11 +33,11 @@ curlimages/curl rancher/curlimages-curl 7.70.0
 curlimages/curl rancher/curlimages-curl 7.73.0
 curlimages/curl rancher/mirrored-curlimages-curl 7.70.0
 curlimages/curl rancher/mirrored-curlimages-curl 7.73.0
+directxman12/k8s-prometheus-adapter rancher/mirrored-directxman12-k8s-prometheus-adapter v0.8.3
 directxman12/k8s-prometheus-adapter-amd64 rancher/directxman12-k8s-prometheus-adapter-amd64 v0.6.0
 directxman12/k8s-prometheus-adapter-amd64 rancher/directxman12-k8s-prometheus-adapter-amd64 v0.7.0
 directxman12/k8s-prometheus-adapter-amd64 rancher/mirrored-directxman12-k8s-prometheus-adapter-amd64 v0.7.0
 directxman12/k8s-prometheus-adapter-amd64 rancher/mirrored-directxman12-k8s-prometheus-adapter-amd64 v0.8.3
-directxman12/k8s-prometheus-adapter rancher/mirrored-directxman12-k8s-prometheus-adapter v0.8.3
```

Comment:
> Change does not remove any existing Images or Tags

I know it says index-list.txt but its supposed to be this file. Can't remove stuff

Reply:
Can you actually change that as part of this PR. It lives here: https://github.com/rancher/image-mirror/blob/master/.github/PULL_REQUEST_TEMPLATE.md

Reply:
Yeah, no problem

Reply:
Just to make sure, this comments is just about updating the template correct? None of my commits remove anything. It might look that way, but it's just moving stuff in the right order before adding new images in the next commit.